examples/braket_features/Using_rigetti_calibration_data_in_noise_channel_parameters/Using_rigetti_calibration_data_in_noise_channel_parameters.ipynb

```diff
@@ -0,0 +1,1151 @@
+{
```

Comment:
<p>Assuming that the reader knows what the T1 value is for? Should introduce it... I see that you define it a little bit later... maybe pull the qubit specs up a bit and intro them, then run the process to get them.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>